service/javadsl/server/src/main/scala/com/lightbend/lagom/internal/javadsl/server/AdditionalRouters.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.javadsl.server
+import java.util
+import java.util.Collections
+
+import play.routing.Router
+
+trait AdditionalRouters {
+  def getRouters: util.List[Router]
+}
+
+class EmptyAdditionalRouters extends AdditionalRouters {
+  override def getRouters: util.List[Router] = Collections.emptyList()
+}
+class NonEmptyAdditionalRouters(val routers: util.List[Router]) extends AdditionalRouters {
+  override def getRouters: util.List[Router] = routers
+}
```

Comment:
If we keep this API as is, it should be rewritten in `java` and moved out of `.internal.`.

See example usage in https://github.com/renatocaval/hello-java-additional-router/blob/7b4646bb1a7eed614b58a4b0a496dc628f8f16a5/hello-impl/src/main/java/com/example/hello/impl/HelloModule.java#L42

Reply:
Yep, doing so (if this indeed remains being kept)

Reply:
Also the ADT style was not really that needed here, went with a single instance for the empty case